Src/FluentAssertions/Specialized/AssemblyAssertions.cs

```diff
@@ -19,7 +19,7 @@ public AssemblyAssertions(Assembly assembly)
             Subject = assembly;
         }
 
-#if NET45 || NET47 || NETSTANDARD2_0 || NETCOREAPP2_0 // TODO :: Should be able to remove based on: https://github.com/dotnet/corefx/issues/1784#issuecomment-218803619
```

Comment:
I tried to remove this, as noted in the TODO.
But I got a lot of errors.
So I recommend to remove now this TODO, because it is not correct (anymore).

Reply:
@dennisdoomen Do you have an opinion to this?

Reply:
If we (still) can't remove, don't